cookbook/workflows/async/employee_recruiter_stream.py

```diff
@@ -0,0 +1,307 @@
+import asyncio
```

Comment:
Maybe add a docstring on top to explain what is showcased here (workflow input with kwargs, etc)